sematic/cli/clean.py

```diff
@@ -0,0 +1,78 @@
+# Standard Library
+import logging
+import os
+from logging.config import dictConfig
+
+# Third-party
+import click
+
+# Sematic
+from sematic.api.endpoints.auth import get_cleaner_api_key
+
+# TODO: will use API client to perform cleaning operations
+# import sematic.api_client as api_client
+from sematic.cli.cli import cli
+from sematic.config.user_settings import UserSettingsVar
+from sematic.logging import make_log_config
+
+logger = logging.getLogger(__name__)
+
+
+@cli.command("clean", short_help="Clean up orphaned objects")
+@click.option(
+    "--orphaned-jobs",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Clean jobs whose associated runs have stopped.",
+)
+@click.option(
+    "--orphaned-resources",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Clean up external resources (ex: Ray clusters) whose runs have stopped.",
+)
+def clean(orphaned_jobs: bool, orphaned_resources: bool):
+    """
+    Clean up objects that are no longer needed.
+    """
+    running_as_cron_job = os.environ.get("RUNNING_AS_CRON_JOB", None) is not None
+    if running_as_cron_job:
+        dictConfig(make_log_config(log_to_disk=False))
+        echo = logger.info  # type: ignore
+        api_key = get_cleaner_api_key()
+        os.environ[UserSettingsVar.SEMATIC_API_KEY.value] = api_key
+        echo("Starting cron cleaner from cron job")
```

Comment:
The cleaner job should be automatically started as a pod by the deployment instead of needing manual intervention.
The manual CLI cleaner should only do a one-shot cleaning in case of emergency.

Reply:
I agree with this, but I think the point is that this CLI command will be called by the cron job itself if I am not mistaken.

Reply:
Is it possible to start the cron job pod through a different entry point, so that we completely avoid the risk of a user mistakenly submitting multiple cron jobs?

Reply:
The intended behavior is:
- when you deploy Sematic, a cron job is automatically created which invokes the `./sematic clean` CLI periodically
- if you are running locally, you can manually clean using the CLI (most of the things that would need to be cleaned aren't relevant locally, but there will be some things, like runs whose resolutions died but the runs are still in the running state).
- if you are an admin, and there is some problem with the k8s cron, you can manually run a clean using the CLI